easy_setup/install_ext_software.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Run this script after setting up TRAL by using setupTRAL.sh.
+# This script will automatically called within setupTRAL or can be executed afterwards.
+# You may need superuser privileges.
+
+######################
+### Prepare Filesystem
+
+#provide paths from config file (has to be in the same directory than setupTRAL.sh)
+. configTRAL_path.cfg
+
+
+######################
+### Installing external software
+
+
+install_ext_software () {
+    
```

Comment:
It would be good to print the license terms for each external software before accepting. That could go in a later PR though.

Reply:
I put it into my ToDoes.